cli/cli.go

```diff
@@ -369,6 +375,85 @@ func getMigrateCommand() cli.Command {
 	}
 }
 
+func getCreateInitialMigrationCommand() cli.Command {
+	return cli.Command{
+		Name:        "create",
```

Comment:
Does `gohan migrate create` always generate initial migration? I think, it should be called `gohan migrate initial`, it leaves some space for diff subcommand. Goose file with some alter tables is a good start.